index.js

```diff
@@ -142,6 +143,20 @@ class Dynastar {
 
     return ret;
   }
+
+  /**
+   * Removes attributes that are created by Global Tables and should never be written.
+   * @see https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/globaltables_HowItWorks.html
+   * @param {Object} data The model data
+   * @returns {Object} The model data without the global-table attributes
+   */
+  _omitGlobalTableData(data) {
+    return omit(data, [
```

Comment:
nitpick `delete data['aws:rep:deleting']`? No need to change, just wondering what `omit` provides?

Reply:
oh good point. let me look. i just copied over stuff from bffs.  I believe this doesn't affect the initial object like `delete` would, but I want to double check that.

Reply:
updated the new tests to show the initial spec is unchanged.

Reply:
Though I supposed I could have done the same still by using delete over the newly created object. :|

Reply:
There I fixed it. lodash.omit is no more. Thanks :)